src/Generics/Linear/TH.hs

```diff
@@ -569,5 +570,6 @@ Why do we do this? Imagine what the instance would be without the explicit retur
   instance Generic1 S where
     type Rep1 S = ... (Rec0 k)
 
-This is an error, since the variable k is now out-of-scope!
```

Comment:
A few lines above it still says "return kind", because I missed it. Could you remove the mysterious "return"?

Reply:
I've changed "explicit return kind" to "explicit kind annotation".